referenceframe/model.go

```diff
@@ -274,9 +274,6 @@ func (m *SimpleModel) UnmarshalJSON(data []byte) error {
 		frameName = ser.Model.Name
 	}
 
-	m.baseFrame = &baseFrame{name: frameName, limits: ser.Limits}
```

Comment:
I'm pretty sure you only moved these up because you were observing nil pointer dereferences in the `m.limits = []Limit{}` assignment in `SetOrdTransforms` [here](https://github.com/viamrobotics/rdk/blob/main/referenceframe/model.go#L237).